tests/scripts/setup_test

```diff
@@ -0,0 +1,63 @@
+#! /usr/bin/env bash
+set -e
+
+# remove tls cert for oidc login
+FINGERPRINT=$(lxc config trust list | grep lxd-ui.crt | awk '{print $8}')
+lxc config trust remove ${FINGERPRINT}
+
+# setup oidc configs
+if [ -f .env.local ]
+then
+  set -o allexport; source .env.local; set +o allexport
+fi
+lxc config set oidc.issuer=${LXD_OIDC_ISSUER}
+lxc config set oidc.client.id=${LXD_OIDC_CLIENT_ID}
+lxc config set oidc.audience=${LXD_OIDC_AUDIENCE}
+
+# create oidc user admin, this will be the account for login during testing
+lxd sql global "
+    INSERT OR REPLACE INTO identities 
+    (id, auth_method, type, identifier, name, metadata) 
+    VALUES 
+    (
+        (SELECT id from identities WHERE name='admin'),
+        2, 
+        5, 
+        'admin@admin.com', 
+        'admin', 
+        '{}'
+    );
+"
```

Comment:
Do we still need this? Can we log in with that user normally now and let LXD create the user?

We can setup the idp group mapping and a group with admin permissions to drive the permission allocation.

Reply:
No, we don't need this but I just thought this is easier to setup and results in one less dependency to the auth0 account. But we can get it setup this way for sure.